### DISCUSSION TITLE
Question: Is the Resources\_gen Directory required?

### DISCUSSION THREAD

**Discussion by MatthewMcD** [onweru/newsroom#33] [Q&A] [answered]
Thank you for this theme!

Under the example site there is a directory: `exampleSite\resources\_gen\assets\sass\sass`

Is it necessary or is it generated in the build?

Thanks!

**Comment by onweru** [comment, marked as answer]
Hi @MatthewMcD, 

Yes. It's [a requirement](https://github.com/gohugoio/hugoThemes#resources) by the Hugo themes team.  For your hosting setup though, it's not necessary.

**Reply by MatthewMcD** [reply]
Thanks, I was asking because of the saas\saas structure.

Great job on this theme. I was up and running in minutes. Now to build the site. :)

**Reply by onweru** [reply]
You're most welcome. I'm glad you find the theme helpful
